app/screens/home/channel_list/categories_list/header/loading_unreads.tsx

```diff
@@ -52,17 +53,6 @@ const LoadingUnreads = () => {
         };
     }, [loading]);
 
-    useEffect(() => {
-        const listener = DeviceEventEmitter.addListener(Events.FETCHING_POSTS, (value: boolean) => {
-            setLoading(value);
-            if (value) {
-                rotation.value = 0;
```

Comment:
we don't want to reset the rotation to 0 degrees?

Reply:
I have done some tests, and it seems the only difference is where the loading spinner starts. It keeps going the full circle. I can set it back to 0 in the effect we already have either when it starts (loading changes to true) or when it stops (loading changes to false).